src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -218,6 +321,8 @@ private function getRelationDetailsFromInput($input)
             // we make sure that at least once we set the relation details
             $fieldDetails = Arr::get($relationDetails, 'relations.'.$key, []);
             $fieldDetails['model'] = $fieldDetails['model'] ?? $field['model'];
+            $fieldDetails['fallback_id'] = $fieldDetails['fallback_id'] ?? $field['fallback_id'] ?? false;
+            $fieldDetails['force_delete'] = $fieldDetails['force_delete'] ?? $field['force_delete'] ?? true;
```

Comment:
Ugh... do we really have to add these attributes for ALL relationships, when they're only used by two of them? Then again, this is one place, if we add them to the fields that use them... it'd be 3 places right?... 😔

Reply:
Right right right!!! If this is only ever going to be used inside the `relationship` field... where we already have a switchboard depending on the relation type... let's set `force_delete` and `fallback_id` there, inside the switchboard.

That way, the rest of the relationships fields won't have these `force_delete` and `fallback_id` attributes, which would do NOTHING for them anyway. No more bloat for them 🎉

Right @pxpm ?